it/controllers/pensions/paymentsIntoOverseasPensions/PaymentIntoPensionSchemeControllerISpec.scala

```diff
@@ -438,154 +425,64 @@ class PaymentIntoPensionSchemeControllerISpec
       }
     }
     "submitted" should {
+      "redirect to the expected page" when {
+        "the user has no stored session data at all" in {
```

Comment:
Note that I decided to collapse the four user config variations into one, as those are irrelevant for many of the scenarios.

I've done this across the board.